lang/semgrep-grammars/src/semgrep-c-sharp-pro/grammar.js

```diff
@@ -0,0 +1,145 @@
+/*
+ * semgrep-c-sharp-pro
+ *
+ * Extend the original tree-sitter C# grammar with semgrep-specific constructs
+ * used to represent semgrep patterns.
+ *
+ * c-sharp-pro is the same as c-sharp but tracks a more recent version
```

Comment:
it's a bit annoying that we have to duplicate grammar.js ... but maybe in the longer term it's better.